### ISSUE TITLE
Bug: software won't start

### ISSUE THREAD

**Issue by NithyaMenon** [ferdium/ferdium-app#898]
### Avoid duplicates

- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues) for a bug report similar to mine, in vain

### Ferdium Version

6.2.0 and onwards

### What Operating System are you using?

Windows

### Operating System Version

Windows 10 Home 21H1

### What arch are you using?

x64

### Last Known Working Ferdium version

6.2.3 nightly 11 portable (but this has stopped working)

### Expected Behavior

The app should open

### Actual Behavior

Same behavior that was seen related to Sentry issues, but 3 instances of Ferdium open in my task manager and the app never launches. I have tried on several versions of Ferdium, uninstalling in between each try. 

### Steps to reproduce

Download an executable (portable or auto) 
Open the executable
App installs, but does not open
Every time you click on the app, 3 more instances of Ferdium get added to Task manager

**Comment by SpecialAro** [comment]
@NithyaMenon did you solved the issue (as you marked it as closed)?

What happened?

**Comment by NithyaMenon** [comment]
Hi @SpecialAro yeah sorry I didn't have a chance to leave a proper comment but wanted to just mark that it was fixed. Basically I think the app data got corrupted? So I started my computer today and Ferdium wasn't opening (no update or anything). And then when I went to try a bunch of different versions, even uninstalling the program on my computer, there was still lingering data in AppData folders. I cleared everything and was able to install a recent nightly. So I guess just a lesson if it stops suddenly, to clear all that data and then try a fresh download!
